src/blockdata/block.rs

```diff
@@ -155,6 +157,61 @@ impl BlockHeader {
     }
 }
 
+/// MWEB Block header
+#[derive(PartialEq, Eq, Clone, Debug)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct MwebBlockHeader {
+    height: i32,
+    output_root: [u8; 32],
+    kernel_root: [u8; 32],
+    leafset_root: [u8; 32],
+    kernel_offset: [u8; 32], // blinding factor
+    stealth_offset: [u8; 32], // blinding factor
+    output_mmr_size: u64,
+    kernel_mmr_size: u64,
+}
+
+impl consensus::Decodable for MwebBlockHeader {
+    #[inline]
+    fn consensus_decode<D: io::Read>(d:D,) -> Result<MwebBlockHeader, consensus::encode::Error>{
+        let mut d = d.take( consensus::encode::MAX_VEC_SIZE as u64);
+        // not sure why, but have to skip this byte which always has value of 130
```

Comment:
Where is the 130 value coming from? Do you have an example of a serialized MWEB block that contains that?

Reply:
I was testing it as part of modified electrs-ltc on Litecoin regtest. I don't know where that value comes from but I needed to skip this byte to make it work.

Reply:
Now block height is decoded properly and consumes right amount of bytes, no more skipping.